src/rgw/rgw_realm_reloader.cc

```diff
@@ -121,6 +121,7 @@ void RGWRealmReloader::reload()
 				   cct->_conf->rgw_enable_quota_threads,
 				   cct->_conf->rgw_run_sync_thread,
 				   cct->_conf.get_val<bool>("rgw_dynamic_resharding"),
+                                   false, // dont run notification thread
```

Comment:
i think this should be `true` instead. `RGWRealmReloader` only runs under `rgw::AppMain`, which isn't used by radosgw-admin. if we initialized the store with `run_notification_thread=true`, we should reload it with the same

Reply:
is the "realm reloader" a separate process?